src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -269,8 +293,10 @@ def get(self, bucket, daemon_name=None):
     def create(self, bucket, uid, zonegroup=None, placement_target=None,
                lock_enabled='false', lock_mode=None,
                lock_retention_period_days=None,
-               lock_retention_period_years=None, daemon_name=None):
+               lock_retention_period_years=None, encryption_state='false',
```

Comment:
After seeing this types of changes in tons of places I feel like we would benefit from having something like [GraphQL](https://graphql.org/learn/). 

Just a rant tho, feel free to resolve this comment xd